packages/firebase_ml_vision/lib/src/barcode_detector.dart

```diff
@@ -288,6 +289,11 @@ class Barcode {
   /// Could be null if the bounding rectangle can not be determined.
   final Rect? boundingBox;
 
+  /// Barcode bytes as they were encoded in the barcode.
+  ///
+  /// Null if nothing found.
+  final Uint8List rawBytes;
```

Comment:
The comment suggests that the variable is null, but the type says that it isn't. What should it be?

Reply:
Fixed :)